JavaFX QuantumRenderer: no suitable pipeline found

When trying to run the app on an M1 Mac, I come across the following error:

```
Caused by: java.lang.RuntimeException: Error initializing QuantumRenderer: no suitable pipeline found
```

and the app fails to start up.

Seems to be some issue with looking up the required dependencies.

--- a/build.gradle
+++ b/build.gradle
@@ -4,6 +4,8 @@ plugins {
     id 'com.github.johnrengelman.shadow' version '7.1.2'
     id 'application'
     id 'jacoco'
+
+    id 'org.openjfx.javafxplugin' version '0.0.13'
 }
 
 mainClassName = 'seedu.address.Main'
@@ -18,6 +20,11 @@ repositories {
 
 checkstyle {
     toolVersion = '10.2'
+}
+
+javafx {
+    version = '17.0.6'
+    modules = ['javafx.controls', 'javafx.fxml']
 }
 
 test {

